File: java/src/main/java/gov/doc/ntia/sigmf/examples/callibration/CallibrationAnnotationExample.java
package gov.doc.ntia.sigmf.examples.callibration;

import gov.doc.ntia.sigmf.Annotation;
import gov.doc.ntia.sigmf.Global;
import gov.doc.ntia.sigmf.MetaDoc;
import gov.doc.ntia.sigmf.examples.Example;
import gov.doc.ntia.sigmf.ext.annotation.sensor.CalibrationAnnotation;
import java.util.ArrayList;
import java.util.List;

public class CallibrationAnnotationExample implements Example {

  @Override
  public MetaDoc getExample() {
    Global global = new Global();
    CalibrationAnnotation cal = new CalibrationAnnotation();
    cal.setGainSigan(2.0);
    cal.setNoiseFigureSigan(1.0);
    cal.setOneDbCompressionPointSigan(12.0);
    cal.setEnbwSigan(2.0);
    cal.setGainPreselector(6.0);
    cal.setNoiseFigureSensor(5.0);
    cal.setOneDbCompressionPointSensor(20.0);
    cal.setEnbwSensor(3.0);
    cal.setMeanNoisePowerSensor(5.0);
    cal.setSampleCount(100L);
    cal.setSampleStart(0L);
    List<Annotation> annotations = new ArrayList<>();
    annotations.add(cal);
    MetaDoc metaDoc = new MetaDoc();
    metaDoc.setAnnotations(annotations);
    return metaDoc;
  }
}

File: Csharp/sigmf/sigmf/examples/callibration/CallibrationAnnotationExample.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace sigmf.examples.callibration
{
    public class CallibrationAnnotationExample : Example
    {
        public override MetaDoc getExample()
        {
            MetaDoc metaDoc = new MetaDoc();
            Globals globals = new Globals();
            List<Annotation> annotations = new List<Annotation>();
            CalibrationAnnotation cal = new CalibrationAnnotation();
            cal.GainSigan = 2.0;
            cal.NoiseFigureSigan = 1.0;
            cal.OneDbCompressionPointSigan = 12.0;
            cal.EnbwSigan = 2.0;
            cal.GainPreselector = 6.0;
            cal.NoiseFigureSensor = 5.0;
            cal.OneDbCompressionPointSensor = 20.0;
            cal.EnbwSensor = 3.0;
            cal.MeanNoisePowerSensor = 5.0;
            cal.SampleCount = 1001;
            cal.SampleStart = 01;
            annotations.Add(cal);
            metaDoc.setAnnotations(annotations);
            return metaDoc;
        }
    }
}
